CHANGELOG.md

```diff
@@ -1,16 +1,16 @@
 # Changelog
 All notable changes to this project will be documented in this file.
-
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-
 ## [Unreleased](https://github.com/ACWI-SOGW/well_registry_management/compare/wellregistry-0.1.0...master)
+-   Added Tooltips for all monitoring location input fields
```

Comment:
Please add the ### Added tag in the line before this line.

Reply:
fixed